tracker_automations/continuous_manage_queues/lib.sh

```diff
@@ -389,3 +389,30 @@ function run_C() {
         echo "$BUILD_NUMBER $BUILD_TIMESTAMP ${issue} moved out from current: held" >> "${logfile}"
     done
 }
+
+function run_param_validation() {
+    nowdate=$(date +%Y%m%d)
+    releasedate=$1
+    lastweekdate=${2:-$(date -d "${releasedate} -7day" +%Y-%m-%d)}
```

Comment:
Ah, sorry, forgot to comment as part of the review. What if we do the 2 params mandatory? I mean, if any of them is empty... then return 1 too. That way, we can remove the code (dupe) recalculating the `lastweekdate`. Whoever is calling to the function will have to provide both dates.

For your consideration, I'm not strong on this (just hate dupe code, easy to forget if changes are performed, heh, maybe you need another function to calculate that date, that would be another alternative).

Ciao :-)

Reply:
Yep, good one. This is done.